cpp/test/Ice/logger/Client1.cpp

```diff
@@ -36,7 +36,7 @@ Client1::run(int argc, char** argv)
     }
     test(s.find(programName) != string::npos);
     in.close();
-    remove("log.txt");
+    (void)remove("log.txt");
```

Comment:
can we use `[[maybe_unused]]`?

Reply:
Yes, it can be `(void)` or `[[maybe_unused]] auto _ = `.